ONSdigital/dp-code-list-api
# Add script to convert CSV to JSON for mongo import

### What
-Separated code lists into individual files for easier management
-Added indexing of codelist ID field to separate from mongo _id field
-Created and documented a script to allow converting a CSV file to additional JSON lines for mongo import

### How to review

- Checkout branch
- Find a V4 file with an unimported dimension
- Follow documentation to import the file
- Check documented outputs occur

### Who can review
@Matt-Guest - I probably need to discuss some of these changes with you as I have made very slight changes to mongo structures
 @robchamberspfc - If you want to check this branch out and tell me if the documentation is clear enough/things seem to work?
